src/routes/auth.js

```diff
@@ -18,4 +18,6 @@ router.get(
   authController.saveUserToTokenAndCookie
 );
 
+router.get('/me', authController.protectedEndpoint);
```

Comment:
I already created a route for /profile and wrote the exact code. Is this necessary?